docs/sources/operators-guide/tools/query-tee.md

```diff
@@ -35,17 +35,25 @@ chmod +x query-tee
 ## Configure the query-tee
 
 The query-tee requires the endpoints of the backend Grafana Mimir clusters.
-You can configure the backend endpoints by setting the `-backend.endpoints` flag to a comma-separated list of HTTP or HTTPS URLs.
-For each incoming request, the query-tee clones the request and sends it to each configured backend.
+You can configure the backend endpoints by setting the `-backend.endpoints` flag to a comma-separated list endpoints:
 
-> **Note:** You can configure the query-tee proxy listening port via the `-server.service-port` flag.
+- HTTP endpoints: via HTTP or HTTPS URLs. Example: `http://query-frontend:80`.
+- gRPC endpoints: via gRPC URI scheme. Example: `dns:///query-frontend:9095`.
+  For each incoming request, the query-tee clones the request and sends it to each configured backend.
```

Comment:
Otherwise it's under gRPC endpoints, but it actually applies both to HTTP and gRPC.
```suggestion

For each incoming request, the query-tee clones the request and sends it to each configured backend.
```